src/test/java/com/synopsys/integration/alert/component/certificates/web/CertificateTestUtil.java

```diff
@@ -17,7 +17,8 @@
 import com.synopsys.integration.alert.database.certificates.CustomCertificateRepository;
 import com.synopsys.integration.alert.util.AlertIntegrationTest;
 
-public class CertificateTestUtil extends AlertIntegrationTest {
+@AlertIntegrationTest
+public class CertificateTestUtil {
```

Comment:
Does this class make sense as an AlertIntegrationTest? This is a test utility not a class with tests.